src/main/webapp/resources/js/components/Buttons/AddMemberButton.jsx

```diff
@@ -95,7 +100,7 @@ export function AddMemberButton({
   }, [visible]);
 
   const addMember = () => {
-    addMemberFn({ id: userId, role })
+    addMemberFn({ projectId, id: userId, role })
```

Comment:
That would make no sense, then it would be the same name as the function it is calling it within.  This `addMemberFn` is passed in as a prop to the component.

Reply:
Oops totally missed that. All good